Octokit/Models/Request/MilestoneRequest.cs

```diff
@@ -39,8 +39,10 @@ internal string DebuggerDisplay
 
     public enum MilestoneSort
     {
-        [Parameter(Value = "due_date")]
+        [Parameter(Value = "due_on")]
```

Comment:
I think this might've been a bug:

> What to sort results by. Either `due_on` or `completeness`. Default: `due_on`
> - https://developer.github.com/v3/issues/milestones/